server/controllers/StatusController.js

```diff
@@ -41,7 +40,8 @@ const getStatus = async (req, res) => {
       const mission = latestMission;
       const vehicle = await getVehicle(latestMission.vehicle_id);
       const status = 'in_mission';
-      const currentStatus = _.find(missionProgress, {'status': vehicle.status});
+      const currentStatus = missionProgress[''+vehicle.status];
```

Comment:
Why concatenate an empty string to it? There is no need to cast to string, it is done implicitly

Reply:
ohk. beginners mistake :+1: